src/main/java/com/project/comgle/config/WebSecurityConfig.java

```diff
@@ -59,8 +59,7 @@ public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Excepti
         http.sessionManagement().sessionCreationPolicy(SessionCreationPolicy.STATELESS);
 
         http.authorizeRequests() //  요청에 대한 보안 검사를 구성한다.
-                .antMatchers("/**").permitAll()
-
+                .antMatchers("/login").permitAll()
```

Comment:
Finally, I can understand all about it! thank u :)

Reply:
Thank you sun~ 💯